src/content/docs/en/guides/assets.mdx

```diff
@@ -336,9 +336,9 @@ It returns an object with the following properties:
 
 ## Using sharp
 
-By default, Astro uses [Squoosh](https://github.com/GoogleChromeLabs/squoosh) to transform your images. This supports all JavaScript environments.
+By default, Astro uses [Squoosh](https://github.com/GoogleChromeLabs/squoosh) to transform your images.
 
-If you're deploying to a Node environment, you may want to use [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
+If you're deploying to a Node environment, we recommend using [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
```

Comment:
Is it worth mentioning static sites separately? The “Node environment” is specifically relevant to SSR deployments I guess (as you _always_ need a Node environment for the initial build).
```suggestion
If you're building a static site or deploying an SSR site to a Node environment, we recommend using [sharp](https://github.com/lovell/sharp), which offers faster performance but requires Node. To use sharp, first install it:
```